### ISSUE TITLE
[docs] "Gatsby Brand Guidelines" link goes to home page

### ISSUE THREAD

**Issue by anuraghazra** [gatsbyjs/gatsby#19685]
## Description

https://www.gatsbyjs.org/contributing/submit-to-plugin-library page's **next link ([Gatsby brand Identity](https://www.gatsbyjs.org/guidelines/color/))** goes to home page.

### Steps to reproduce

- Goto https://www.gatsbyjs.org/contributing/submit-to-plugin-library/
- Scroll To Bottom
- Click On The **"Gatsby Brand Guidelines ->"** link
  ![gatsby next link](https://user-images.githubusercontent.com/35374649/69342128-e54bde80-0c90-11ea-870d-f81f904b1834.PNG)

### Expected result

Should go to https://www.gatsbyjs.org/guidelines/color/ page

### Actual result

Goes to https://www.gatsbyjs.org page

### Environment

Run `gatsby info --clipboard` in your project directory and paste the output here.

**Comment by lannonbr** [comment]
It looks like this is because the Brand Guidelines section doesn't actually have a main page. I would have to look into how the pagination is, but maybe a possible solution is to link to the first actual page in the brand guidelines which would be the "color" page discussing the color system on .org

**Comment by anuraghazra** [comment]
Yes! and i new to gatsby's source code but i think it's related to this line of code. after `title` we can add a `link` property.

https://github.com/gatsbyjs/gatsby/blob/958cf1fcf89405c3946d59a315afaa8ab3819622/www/src/data/sidebars/contributing-links.yaml#L72

**Comment by lannonbr** [comment]
So I took a deeper look into this topic, and due to the fact that the "Gatsby Brand Guidelines" doesn't have a `link` field in the contributing links yaml file as @anuraghazra commented on, it will jump over to the homepage.

That said, the brand guideline pages do not use the normal docs markdown template and therefore even if we skipped into the first page, there's no sidebar, no prev / next links, so it's a completely different feeling when you jump into this section. With this, I'm unsure if the flow of having this as if it is a normal portion of the contributing docs at the moment feels right.

@gatsbyjs/learning, This seems like something to be further researched on how to solve rather than just fixing the wrong navigation bug that exists now.
